client/galaxy/scripts/mvc/dataset/dataset-edit-attributes.js

```diff
@@ -0,0 +1,250 @@
+
+define( [ 'utils/utils', 'mvc/ui/ui-tabs', 'mvc/ui/ui-misc', 'mvc/form/form-view' ], function( Utils, Tabs, Ui, Form ) {
+
+    /** Dataset edit attributes view */
+    var View = Backbone.View.extend({
+
+        initialize: function( dataset_id ) {
```

Comment:
Instead of just parsing `dataset_id` I suggest to parse a regular dictionary with a `dataset_id` attribute.